classes/movie.rb

```diff
@@ -0,0 +1,24 @@
+require 'json'
+require_relative 'helper'
+require_relative '../app/item'
+
+class Movie < Item
```

Comment:
- [ ] can_be_archived?() in the Movie class should override the method from the parent class. It should return `true` if the parent's method returns `true` OR if silent equals `true`, otherwise, it should return false. This should be implemented.